2.5/test/run

```diff
@@ -179,8 +191,15 @@ for server in ${WEB_SERVERS[@]}; do
   ct_npm_works
   check_result $?
 
+  test_from_dockerfile
+  check_result $?
+
+  test_from_dockerfile_s2i
```

Comment:
```suggestion
  test_from_dockerfile s2i
```

Reply:
@fila43 still, could we adopt this, together with [above suggestion](https://github.com/sclorg/s2i-ruby-container/pull/295#discussion_r509980772), and therefore remove duplicate code? (Which is the reason I did begin this conversation.)

Reply:
**EDIT:** [this](https://github.com/sclorg/s2i-ruby-container/pull/295#discussion_r509980459) is actually the core part of the change I'm suggesting.